misc/python/materialize/mzbuild.py

```diff
@@ -223,6 +233,13 @@ def __init__(self, rd: RepositoryDetails, path: Path, config: Dict[str, Any]):
         self.strip = config.pop("strip", True)
         self.extract = config.pop("extract", {})
         self.rustflags = config.pop("rustflags", "")
+        if rd.coverage:
```

Comment:
This does not seem to bring in the nightly compiler. If I run

```
TD_TEST=catalog.td ./mzcompose  --mz-coverage run testdrive
```

I get:

```
$ /home/pstoev/benesch-mzcompose-env/bin/xcompile build --bin materialized --release
error: failed to run `rustc` to learn about target-specific information

Caused by:
  process didn't exit successfully: `rustc - --crate-name ___ --print=file-names -Zinstrument-coverage -Clink-arg=-Wl,--warn-unresolved-symbols --target x86_64-unknown-linux-gnu --crate-type bin --crate-type rlib --crate-type dylib --crate-type cdylib --crate-type staticlib --crate-type proc-macro --print=sysroot --print=cfg` (exit code: 1)
  --- stderr
  error: the option `Z` is only accepted on the nightly compiler
```

calling ```ci-builder``` causes the proper compilation to happen.